src/Helpers/Portfolio/PortfolioHelper.js

```diff
@@ -15,16 +15,16 @@ export function listPortfolioItems() {
   return userApi.listPortfolioItems();
 }
 
-export function getPortfolioItem(portfolioId, portfolioItemId) {
-  return userApi.fetchPortfolioItemFromPortfolio(portfolioId, portfolioItemId);
+export function getPortfolioItem(portfolioItemId) {
+  return fetch(`${SERVICE_PORTAL_API_BASE}/portfolio_items/${portfolioItemId}`).then(data => data.json());
```

Comment:
Are we switching to fetch here because of any issues with the openapi client?

Reply:
No issue the client does not pass every attribute (Same as with the pagination). This is temporary until that is resolved.

Reply:
Sounds good, thx